### PR TITLE
Image upload for admins

### PR DESCRIPTION
**Pull request by jbarrett**
Cc: @tagawa 

Review notes:
- Image upload link should be available from admin index (/admin)
- Only admins can upload
- Only image files can be uploaded
  - I renamed some random text file to have a jpg extension to verify this
  - This is only loosely related to this PR - this feature was implemented separately aeons ago.

### CODE REVIEW

**Comment by tagawa** [comment]
Cool, this will be good to have - thanks.

**Comment by MariagraziaAlastra** [comment]
What's a good set of tests to perform for this?

**Comment by jbarrett** [comment]
@MariagraziaAlastra Calling this WIP until I have tests.

File: t/image_upload_basics.t [hunk 1]
```diff
@@ -0,0 +1,108 @@
+use strict;
+use warnings;
+
+# Database setup
+BEGIN {
+    unlink 'ddgc_test.db';
+    $ENV{DDGC_DB_DSN} = 'dbi:SQLite:dbname=ddgc_test.db';
```
Comment (MariagraziaAlastra) [inline t/image_upload_basics.t]: This sets an env variable, right? Extremely useful!

**Comment by MariagraziaAlastra** [comment]
@jbarrett LGTM! I ran the test file and I also tested it manually:
screenshot-maria duckduckgo com 5001 2016-04-20 15-12-57


Deeheehee. Thanks!

**Comment by tagawa** [comment]
You platform folks are fast :smiley:

File: t/image_upload_basics.t (hunk 1 above)
Comment (jbarrett) [inline t/image_upload_basics.t]: Yeah, anything you put in a `BEGIN` block is done first so you can set up the environment and such before any other code runs.

**Pull request merged**